docs/agents/sematext-agent/events.md

```diff
@@ -0,0 +1,63 @@
+title: Sematext Agents Events
+
+Below you can find a list of all the events sent from Sematext Agent
+
+## Agent started
+
+This event is sent when the agent is started.
+
+Message: agent_started sematext-agent `version` / on `hostname`
+
+## Agent stopped
+
+This event is sent when the agent is stopped.
+
+Message: agent_stopped sematext-agent `version` / on `hostname`
+
+## Out of memory (OOM)
+
+This event is sent when a process is terminated by OOM Killer.
```

Comment:
Maybe we want to explicitly say that this is about any Linux process, not agent process

Reply:
I see (Linux only) below.  If this is Linux only as well, we should add that.

Reply:
added Linux Events section.